rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/jobs/JobRequest.groovy

```diff
@@ -27,6 +27,7 @@ import groovy.transform.Immutable
 class JobRequest {
   List<String> tokenizedCommand
   List<String> maskedPackerParameters = []
+  String jobId = ""
```

Comment:
Is there a reason to initialize this to an empty string? I'd think there isn't any path hat has `JobRequest` being constructed without `jobId` being passed as a named constructor arg.